### PR TITLE
Pop up menu implement

### PR DESCRIPTION
**Pull request by alexansaa**
This milestone has implemented the next features:
- Mobile version popup main menu implementation
- Media queries breakpoints complying pop-up main menu
- Clicks on Links go to their respective sections
- Close and open buttons for the main popup menu
- The popup closes when clicked on close button or on links
- Only popup mobile version has been implemented so far

### CODE REVIEW

File: myStyles.css [hunk 1]
```diff
@@ -426,10 +479,44 @@
...
+  }
+
+  .menuTextContainer a {
+    text-decoration: none;
+    color: #fff;
+    font-family: "Inter", sans-serif;
+    font-size: 40px;
+    text-align: left;
+    padding-left: 10%;
+  }
```
Comment (AbelG101) [inline myStyles.css:503]: - [ ] Ouch! this is close but please make sure to use the proper color, font weight, and spacing between the option menus as there seems to be a bit of discrepancy between your design and the Figma design as shown below. As a front-end developer, you have to pay attention to small details like this as they provided a polished UI.
- Figma design:
  image
- Your design:
  image
- You can simply achieve this by adding a `font-weight: bold`, a color of `#fff5e1`, and some vertical spacing between the menu options. ✔

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @alexansaa 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Menu opens and closes as expected ✔
- Menu option redirects to the proper section ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: myStyles.css [hunk 2]
```diff
@@ -173,10 +183,47 @@
...
   .menuTextContainer {
     display: none;
+    flex-direction: column;
+    position: fixed;
+    left: 0;
+    right: 0;
+    top: 0;
+    bottom: 0;
+    z-index: 10;
+    background-color: rgb(69, 67, 67);
```
Comment (Kossi-stack) [inline myStyles.css]: - [ ] Kindly make sure the background color meets the requirement `#3C3A39` as well as the text color
Screenshot 2023-06-21 at 10 32 58

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @alexansaa,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ Linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @alexansaa, 

To highlight 👍🏽 
✔️ App runs smoothly
✔️ App matches the design

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**